apps/bfd-data-npi/src/test/java/gov/cms/bfd/data/npi/lookup/NPIOrgLookupTest.java

```diff
@@ -66,8 +62,47 @@ public void shouldNotReturnWhenNPINumberIsEmptyAndWhenConstructorSetToTrue() thr
   @Test
   public void shouldNotReturnWhenNPINumberIEmptyStringAndWhenConstructorSetToTrue()
       throws IOException {
-    NPIOrgLookup npiOrgDataLookup = NPIOrgLookup.createNpiOrgLookupForTesting();
-    Optional<String> npiOrgDisplay = npiOrgDataLookup.retrieveNPIOrgDisplay(Optional.of(""));
+    npiOrgDisplay = npiOrgDataLookup.retrieveNPIOrgDisplay(Optional.of(""));
     assertEquals(false, npiOrgDisplay.isPresent());
   }
+
+  /** Should Return Map When Input Stream Is Formatted Correctly. */
```

Comment:
Ideally the description should be a little more than the name. What does "formatted correctly" mean here?

Reply:
Done